test_app/settings.py

```diff
@@ -185,6 +185,7 @@
 ANSIBLE_BASE_JWT_MANAGED_ROLES.append("System Auditor")  # noqa: F821 this is set by dynamic settings for jwt_consumer
 ANSIBLE_BASE_ALLOW_SINGLETON_USER_ROLES = True
 ANSIBLE_BASE_ALLOW_SINGLETON_TEAM_ROLES = True
+ANSIBLE_BASE_OAUTH2_PROVIDER_PERMISSIONS_CHECK_IGNORED_VIEWS = ["drf_spectacular.views.SpectacularSwaggerView"]
```

Comment:
Should this be added by default in the dynamic settings? That way if anything implements dab_api_documentation they "get it for free"?

Reply:
I left this ignore in for testing purposes (to reach the ignore code path), but the defaults should not need it for the view to pass.